resources/maskinporten_clients.py

```diff
@@ -66,19 +67,29 @@ def create_client(
         )
     )
     try:
-        new_client = MaskinportenClient(body.env).create_client(body).json()
+        maskinporten_client = MaskinportenClient(body.env)
     except UnsupportedEnvironmentError as e:
         raise ErrorResponse(status.HTTP_400_BAD_REQUEST, str(e))
 
-    # TODO: Roll back created client if permission creation fails.
-    create_okdata_permissions(
-        resource_name=f"okdata:maskinporten-client:{body.env}-{new_client['client_id']}",
-        owner_principal_id=auth_info.principal_id,
-        auth_header=service_client.authorization_header,
-    )
+    new_client = maskinporten_client.create_client(body).json()
+    new_client_id = new_client["client_id"]
+
+    try:
+        create_okdata_permissions(
+            resource_name=f"okdata:maskinporten-client:{body.env}-{new_client_id}",
+            owner_principal_id=auth_info.principal_id,
+            auth_header=service_client.authorization_header,
+        )
+    except requests.RequestException as e:
+        # Permission creation failed. Retract the created Maskinporten client.
+        log_exception(e)
```

Comment:
Passer vi på å ikke logge noe sensitivt mtp. ROS ? :P

Reply:
Her får man ut tracebacken for excpetionen fra `requests`, så tror (og håper) ikke det. :sweat_smile: